src/Nest/Cluster/TaskManagement/GetTask/TaskInfo.cs

```diff
@@ -36,5 +36,8 @@ public class TaskInfo
 
 		[JsonProperty("cancellable")]
 		public bool Cancellable { get; internal set; }
+
+		[JsonProperty("headers")]
+		public IReadOnlyDictionary<string, string> Headers { get; internal set; } = EmptyReadOnly<string, string>.Dictionary;
 	}
```

Comment:
missing child tasks in `children` array?

Reply:
Good catch, have added.